.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
```

Comment:
What reads .editorconfig?

Reply:
Currently, only IDE will read `.editorconfig`, but I'd hope this project will eventually adapt for example [ktlint](https://github.com/pinterest/ktlint/) which would be another example of a tool that reads `.editorconfig` - [docs](https://pinterest.github.io/ktlint/0.49.1/rules/configuration-ktlint/)

Reply:
ktlint is great. I usually use it in combination with detekt-formatting (like what I did in [Mavericks](https://github.com/airbnb/mavericks/pull/191). If you'd like to take that on next, feel free!